lib/task_bunny/manual_queue_supervisor.ex

```diff
@@ -0,0 +1,74 @@
+defmodule TaskBunny.ManualQueues do
+  @moduledoc """
+  Supervisor to allow for only Specific queues to be started manually.
+
+  This is useful when multiple applications inside of an umbrella application
+  use task bunny, you need to launch only the queues specific to a given application
+
+  It supervises Connection and WorkerSupervisor with one_for_all strategy.
+  When Connection crashes it restarts all Worker processes through WorkerSupervisor
+  so workers can always use a re-established connection.
+
+  You can call this supervisor explicitly like this:
+    children = [
+      supervisor(TaskBunny.ManualQueues, [[queues: ["myapp.my.queue", "myapp.my.queue_1"]]])
+    ]
+    Supervisor.start_link(children, [strategy: :one_for_one, name: MyApp.Supervisor])
+
+  Make sure that task_bunny is not a runtime dependency to disable automatic application start. i.e.
+    defp deps do
+      [
+        {:task_bunny, "~> 0.3.2", runtime: false}
+      ]
+    end
+  """
+  use Supervisor
+  alias TaskBunny.{Connection, Config, Initializer, WorkerSupervisor, PublisherWorker}
+
+  @doc false
+  @spec start_link(atom, atom) :: {:ok, pid} | {:error, term}
```

Comment:
@spencerdcarlson What do you think about modifying the current `TaskBunny.Supervisor` to accept an option that requires queues be started manually? This may work for our use case, but probably isn't ideal for the lib. Let me know what you think.

Reply:
Sounds good. I'll do that :)